docker-compose.yml

```diff
@@ -144,6 +144,7 @@ services:
 
   mail:
     container_name: mail
+    restart: unless-stopped
```

Comment:
this is unrelated to this PR

Reply:
removed and will be placed on another pr to itself at a later time.